deploy/gke/recon-prod.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 Google LLC
```

Comment:
do we need recon-autopush?

Reply:
Yep, we do need that. Have some issue on deploying it. Will send a PR when it's ready.